prow/plugins.yaml

```diff
@@ -126,6 +126,9 @@ plugins:
   - docs-no-retest
   - blockade
 
+  kubernetes/kubernetes-template-project:
+  - approve
```

Comment:
The approve plugin config defaults to no associated issue required and no implicit author self-approval. Are those defaults what you want?

Reply:
yes I'm fine with that